ironic/drivers/modules/drac/raid.py

```diff
@@ -470,8 +549,8 @@ def _find_configuration(logical_disks, physical_disks):
                                         physical_disks_by_type,
                                         free_space_mb))
             if not result:
-                error_msg = _('failed to find matching physical disks for all '
-                              'logical disks')
+                error_msg = _('failed to find matching physical disks'
```

Comment:
Need to add a space after "disks" on the line above, or this will end up being "disksfor"